src/pages/desktop.mdx

```diff
@@ -0,0 +1,146 @@
+import { Callout } from "nextra/components"
+import { Card, Cards } from "nextra/components"
+import { Tabs } from 'nextra/components'
+import { useEffect } from 'react'
+
+# Golem Desktop Introduction
+
+Golem Desktop provides a native application experience across Windows, Linux, and macOS. Since the application is currently unsigned, please follow the platform-specific instructions below to install it.
+
+<Callout type="info">
+  Please note that the Golem Desktop application is only supported for the open source version of Golem.
+</Callout>
+
+## Download and Installation Instructions
+
+<Tabs items={['Windows', 'macOS', 'Linux']}>
+  <Tabs.Tab>
+    <Tabs items={['Windows Installer']}>
+      <Tabs.Tab>
+        1. Download the `.exe` installer from the [release page](https://github.com/golemcloud/golem-cli/actions/runs/13761723238/artifacts/2721155019)
+        2. When you run the installer, Windows may show a SmartScreen warning
+        3. Click "More info" and then "Run anyway" to proceed with installation
+        4. Follow the on-screen instructions to complete installation
+      </Tabs.Tab>
+    </Tabs>
+  </Tabs.Tab>
+  <Tabs.Tab>
+    <Tabs items={['Intel Mac', 'Apple Silicon (M-Chip)']}>
+      <Tabs.Tab>
+        1. Download the Intel x64 `.dmg` file from the [release page](https://github.com/golemcloud/golem-cli/actions/runs/13761723238/artifacts/2721155019)
+        2. Open the downloaded `.dmg` file
+        3. Drag the application to your Applications folder
+        4. When first launching, macOS may block the app with a message "cannot be opened because the developer cannot be verified"
+        5. To open the app:
+           - Right-click (or Control-click) on the app icon and select "Open"
+           - Click "Open" in the dialog box that appears
```

Comment:
And related to that, with newer versions you cannot simply force open the app, i think it need some commands to make it work.

Reply:
sudo xattr -r -d com.apple.quarantine /Applications/Golem.app
this works for me

Reply:
you can fully do it from UI (I did not know about the xattr thing). After it fails and tries to put your things to trash, you open the privacy & security settings page, scroll down a lot, and there will be an allow button :). Then on next attempt it asks for password and after that it works.

Reply:
I only see DMGs in the release

Reply:
@vigoo depends on many things :D Ofc i know that, but for me the current latest DMG did not offered that, rather went into the "thrash" route